### ISSUE TITLE
Logging documentation references obsoleted behavior

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3935]
|[<img alt="djfroofy's avatar" src="https://avatars.githubusercontent.com/u/89760?s=50" width="50" height="50">](https://github.com/djfroofy)| @djfroofy reported|
|-|-|
|Trac ID|trac#3935|
|Type|defect|
|Created|2009-07-28 03:36:52Z|
|Branch|https://github.com/twisted/twisted/tree/logging-doc-3935|

Logging documentation (http://twistedmatrix.com/projects/core/documentation/howto/logging.html) contains the following warning:

# Never raise an exception from a log observer. If your log observer raises an exception, it will be removed.

This is no longer true in version 8.2.0 per fix for [1096].

<details><summary>Searchable metadata</summary>

```
trac-id__3935 3935
type__defect defect
reporter__djfroofy djfroofy
priority__low low
milestone__ 
branch__branches_logging_doc_3935 branches/logging-doc-3935
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1248752212000000 1248752212000000
changetime__1264053445000000 1264053445000000
version__None None
owner__ 
cc__jesstess cc__khorn
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess commented|
|-|-|

(In [28033]) Branching to 'logging-doc-3935'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [28034]) Remove old warning about raising exceptions from log observers.

refs [#3935](https://github.com/twisted/twisted/issues/3935)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess removed owner|
|-|-|

I'm not sure how the change set in the ticket description relates to this not being a problem anymore, but I didn't see any comments about exceptions in log.py so I assume it's been fixed.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_5"></a>|@khorn commented|
|-|-|

I think the original poster meant: [#10709](https://github.com/twisted/twisted/issues/10709)

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn set owner to @jesstess|
|-|-|

Also, your change looks fine...assuming the problem really is fixed, I think you can go ahead and merge this.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess set status to `closed`|
|-|-|

(In [28046]) Merge logging-doc-3935

Author: jesstess
Reviewer: khorn
Fixes: [#3935](https://github.com/twisted/twisted/issues/3935)

Remove old warning about raising exceptions from log observers getting
the observers removed as the issue has been fixed since [#10709](https://github.com/twisted/twisted/issues/10709).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
